.gitignore

```diff
@@ -5,3 +5,4 @@ redis.*
 test.log
 settings.json
 .phpunit.result.cache
+dump.rdb
```

Comment:
why this addition?

Reply:
Maybe he wants to prevent it from being tracked by git since it contains a snapshot of the Redis database that is created by Redis when it is executed in a specific environment.